frontend/src/ui/Blocks/Series.tsx

```diff
@@ -193,46 +218,412 @@ type SeriesBlockContainerProps = {
-        ...title && { paddingTop: 0 },
-        backgroundColor: "var(--grey95)",
-        borderRadius: 10,
-    }}>
-        {/* This is a way to make this fancy title. It's not perfect, but it works fine. */}
-        {title && <div css={{ maxHeight: 50 }}>
-            <WithTooltip tooltip={title}>
-                <h2 css={{
-                    backgroundColor: "var(--accent-color)",
-                    color: "var(--accent-color-bw-contrast)",
-                    padding: "4px 12px",
-                    borderRadius: 10,
-                    transform: "translateY(-50%)",
-                    fontSize: 19,
-                    margin: "0 8px",
-                    lineHeight: 1.3,
-                    ...ellipsisOverflowCss(3),
-                    display: "-webkit-inline-box",
-                }}>{title}</h2>
-            </WithTooltip>
-        </div>}
-        {children}
-    </div>
+type View = "slider" | "gallery" | "list";
+
+type ViewContext = {
+    viewState: View;
+    setViewState: (view: View) => void;
+};
+
+const ViewContext = createContext<ViewContext>({
+    viewState: "gallery",
+    setViewState: () => {},
+});
+
+const SeriesBlockContainer: React.FC<SeriesBlockContainerProps> = ({ title, children }) => {
+    const [viewState, setViewState] = useState<View>("gallery");
+
+    return (
+        <ViewContext.Provider value={{ viewState, setViewState }}>
+            <div css={{
+                marginTop: 24,
+                padding: 12,
+                backgroundColor: "var(--grey95)",
+                borderRadius: 10,
+            }}>
+                <div css={{
+                    display: "flex",
+                    [`@media (max-width: ${VIDEO_GRID_BREAKPOINT}px)`]: {
+                        flexDirection: "column",
+                    },
+                }}>
+                    {title && <div css={{
+                        display: "inline-block",
+                        padding: "8px 12px",
+                    }}>
+                        <h2 css={{
+                            color: "var(--grey20)",
+                            fontSize: 20,
+                            lineHeight: 1.3,
+                        }}>{title}</h2>
+                    </div>}
+                    <div css={{
+                        display: "flex",
+                        alignItems: "center",
+                        alignSelf: "flex-start",
+                        fontSize: 14,
+                        gap: 16,
+                        padding: 5,
+                        [`@media not all and (max-width: ${VIDEO_GRID_BREAKPOINT}px)`]: {
+                            marginLeft: "auto",
+                        },
+                        [`@media (max-width: ${VIDEO_GRID_BREAKPOINT}px)`]: {
+                            width: "100%",
+                            justifyContent: "space-between",
+                            paddingLeft: 11,
+                            paddingRight: 11,
+                        },
```

Comment:
Is it intended that the two buttons have space between them? I would think it looks better if they are both on the right.

So I would remove all of these lines. And add `justifyContent: flex-end` but without media query. The replace the `margin-left` here, the parent div should have `justify-content: space-between`. Unless I'm missing something that works better with less CSS and it looks better IMO.

The only "problem" this is that if the title is missing, the buttons are on the left again. I think I would solve that by just removing `title &&`. So that the `<div><h2 /></div>` (or even just `<h2 />` as mentioned above) is always rendered, but possibly with nothing inside.

Reply:
Yeah I did `space-between` because I found the situation depicted below to look a little awkward when the title is short, but I suppose it's not that bad. I could also just use a lower breakpoint for `flex-direction: column`, or align (or rather "justify") the buttons with the heading on the left on smaller screens (second screenshot). Let me know what you prefer. 

<img width="326" alt="Bildschirm­foto 2023-02-20 um 15 25 29" src="https://user-images.githubusercontent.com/94838646/220133232-182bf81d-2769-4a8e-ac57-4ef01f820c7d.png">


<img width="326" alt="Bildschirm­foto 2023-02-20 um 15 32 22" src="https://user-images.githubusercontent.com/94838646/220134641-db089e88-8b27-4f9c-97db-ba09dc09475b.png">

I believe that `justify-content: flex-end` does not work with `align-self: flex-start` (which I use to align the buttons vertically with the first line of multiline headings), but we can use `margin-left: auto` instead of `justify-content` to keep the buttons on the right.

Reply:
Ah mh, thats fair, it does look weird for small titles :/ 
What about using `flex-wrap: wrap` instead of `flex-direction: column`? Then the second line is only created when there is actually not enough space. Specific suggestion:

- Remove all 5 CSS properties that this review comment is attached to.
- Add `margin-left: auto` to the button container (for all screen sizes)
- For width < breakpoint, add `flex-wrap: wrap` for the outer container (and remove `flex-direction: column`)

For any given title, there is still a screen width where it looks weird (since it juuust wrapped around), but thats better than the previous idea. What do you think?

Reply:
Sure, that works for me!

Reply:
I like how it works now I think. The `justifyContent: "space-between"` on the parent div is not even required anymore.